examples/async/tests/async_test.bal

```diff
@@ -45,8 +45,8 @@ function testExist(string text) returns boolean {
         if (i == 4) {
             continue;
         }
-        string out = string.convert(outputs[i]);
-        if (out.contains(text)) {
+        string out = outputs[i].toString();
+        if (stringutils:contains(out, text)) {
```

Comment:
Since we are going to remove stringutils module, it's better if we can use `lang.string` functions in our examples. I guess we can use `indexOf()` here.

@daneshk WDYT?

Reply:
Fixed in here https://github.com/ballerina-platform/ballerina-distribution/pull/544/commits/535ae373dc7ed68adcd520bb0b467abe1b796b13